doc/user/content/get-started.md

```diff
@@ -73,237 +62,465 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
     ```
 
-1. **Create connections.** Connections describe how to connect and authenticate to an external system you want Materialize to read data from. In this case, we want to [create a connection](/sql/create-connection/#kafka) to a Kafka cluster.
+    ```sql
+    CREATE CLUSTER compute_qck REPLICAS (r1 (SIZE = '2xsmall'));
+    ```
+
+    The `2xsmall` replica size is sufficient for the data ingestion and computation in this getting started scenario. Also, these replicas use only 1 of the [4 credits per hour](https://materialize.com/pricing/) you get with your free trial. Combined with the `xsmall` replica in the pre-installed `default` cluster, which uses 1 credit per hour, you still have 2 hourly credits available.
+
+1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the replicas:
 
     ```sql
-    CREATE CONNECTION kafka_connection TO KAFKA (
-      BROKER 'pkc-n00kk.us-east-1.aws.confluent.cloud:9092',
-      SASL MECHANISMS = 'PLAIN',
-      SASL USERNAME = SECRET kafka_user,
-      SASL PASSWORD = SECRET kafka_password
-    );
+    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ```
+
+    ```noftm
+       cluster   | replica |  size   | ready
+    -------------+---------+---------+-------
+     compute_qck | r1      | 2xsmall | t
+    (2 rows)
     ```
 
-    In addition to Kafka, we need a connection to a Schema Registry that Materialize will use to fetch the schema of our sample data.
+    Once the `r1` replica is ready (`ready=t`), move on to the next step.
+
+## Step 3. Ingest streaming data
+
+Materialize supports streaming data from multiple external sources, including [Kafka](/sql/create-source/kafka/) and [PostgreSQL](/sql/create-source/postgres/). The process for integrating a source typically involves configuring the source's network and creating connection objects in Materialize.
+
+For this guide, you'll keep it simple and use a [built-in load generator](/sql/create-source/load-generator/#auction) that simulates an auction house, where users bid on an ongoing series of auctions.
+
+1. Use the [`CREATE SOURCE`](/sql/create-source/load-generator/) command to create the auction house source:
 
     ```sql
-    CREATE CONNECTION csr_connection TO CONFLUENT SCHEMA REGISTRY (
-      URL 'https://psrc-ko92v.us-east-2.aws.confluent.cloud:443',
-      USERNAME = SECRET csr_user,
-      PASSWORD = SECRET csr_password
-    );
+    CREATE SOURCE auction_house
+      IN CLUSTER ingest_qck
+      FROM LOAD GENERATOR AUCTION (TICK INTERVAL '50ms')
+      FOR ALL TABLES;
     ```
 
-    Once created, these connections can be reused across multiple `CREATE SOURCE` statements.
+    Note that the `IN CLUSTER` clause attaches this source to the existing `ingest_qck` cluster, but it's also possible to create a cluster and replica at the time of source creation using the [`WITH SIZE`](/sql/create-source/load-generator/#with-options) option.
```

Comment:
Maybe format as a Note/Aside since it's not directly actionable in this guide.

Reply:
I would prefer note formatting here and a few other places as well, but I couldn't get it to work. When nested in a step, the note shortcode renders wrong:

<img width="787" alt="Screenshot 2023-04-24 at 4 14 57 PM" src="https://user-images.githubusercontent.com/15893490/234106296-9e4a948d-416d-4c00-a43e-04a00cc2b042.png">

Logged an issue: https://github.com/MaterializeInc/materialize/issues/18933